test/account.c

```diff
@@ -151,3 +151,103 @@ int test_account_uri_complete(void)
 
 	return err;
 }
+
+
+int test_account_stun_uri_complete(void)
+{
+	struct account *acc = NULL;
+	int err = 0;
+	struct sip_addr *addr;
+
+	err = account_alloc(&acc, "\"A\" <sip:A@proxy.com>"
+		";stunuser=bob@bob.com"
+		";stunpass=taj:aa"
+		";stunserver=\"stun:42:19@stunserver.org\"");
```

Comment:
maybe remove password from testcode

Reply:
These are not real passwords, or do you mean something else?